tests/run/test_fstring.pyx

```diff
@@ -1169,9 +1168,11 @@ non-important content
         self.assertEqual(f'{0!=1}', 'True')
         self.assertEqual(f'{0<=1}', 'True')
         self.assertEqual(f'{0>=1}', 'False')
-        self.assertEqual(f'{(x:="5")}', '5')
+        # Walrus not implemented yet, skip
+        #self.assertEqual(f'{(x:="5")}', '5')
         self.assertEqual(x, '5')
```

Comment:
This looks like it should probably fail now.

Reply:
It looks as if the tests never ran due to the `__` in the name of the function. Should I remove them?

Reply:
Fixed